Makefile

```diff
@@ -0,0 +1,4 @@
+CC=gcc
+
+accuraterip-checksum: accuraterip-checksum.c
+	$(CC) -std=c99 -l sndfile -o $@ $<
```

Comment:
Hi Leo,

Thanks for your reply. It was a long time ago, but this was designed to work with [GNU Make](https://www.gnu.org/software/make/).

Regarding question number 2, `make` expects the variables in recipes to contain the double quotes inside its contents AFAIK, and this is also (apparently) true for the automatic variables, like `$@` and `$<`. I searched [here](https://www.gnu.org/software/make/manual/make.html#Variables-in-Recipes) and [here](https://www.gnu.org/software/make/manual/make.html#Automatic-Variables). It does not seem to be related to any variable in `bash` though, as the recipes processing happens before handling these commands to the shell.

Regarding question 3, in the second link you will see that the automatic variable `$<` in this case will expand to the name of the first prerequisite - which is the source file.